bucket/openshot.json

```diff
@@ -14,16 +14,23 @@
         }
     },
     "innosetup": true,
+    "pre_install": "Set-Content \"$dir\\openshot.ps1\" \"Start-Process cmd -ArgumentList @('/c', `\"$dir\\launch-win.bat`\") -WorkingDirectory `\"$dir`\"\"",
```

Comment:
Why do you need cmd?

Reply:
I didn't know that I can run `launch-win.bat` directly with `Start-Process`. Thanks.